syntax/pb_test.go

```diff
@@ -0,0 +1,65 @@
+package syntax
+
+import (
+	"testing"
+
+	"github.com/arr-ai/arrai/rel"
+	"github.com/arr-ai/wbnf/parser"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTransformProtobufToTupleFlow(t *testing.T) {
+	t.Parallel()
+
+	code := "//encoding.proto.decode(//os.file('../translate/pb/test/sysl.pb'), 'Module', " +
+		"//os.file('../translate/pb/test/petshop.pb'))"
+	pc := ParseContext{SourceDir: ".."}
+	ast, err := pc.Parse(parser.NewScanner(code))
+	assert.NoError(t, err)
+
+	codeExpr := pc.CompileExpr(ast)
+	val, err := codeExpr.Eval(rel.EmptyScope)
+
+	assert.NoError(t, err)
+	shop, _ := val.(rel.Tuple)
+	apps, _ := shop.Get("apps")
+
+	assert.NotNil(t, apps)
+}
+
+func TestTransformProtobufToTupleCompareResult(t *testing.T) {
```

Comment:
Nit: put each of these cases in a separate function if they are indeed testing different things, and include a little bit in the function name to describe the case being tested.